src/components/PrivacyModal/PrivacyModal.tsx

```diff
@@ -0,0 +1,114 @@
+import { Modal, Typography, Box, Button, Link } from '@common/components'
+import { FormGroup, FormControlLabel, Checkbox } from '@mui/material'
+import { usePrivacyModal as _usePrivacyModal, PrivacyModalHookReturn } from './PrivacyModal.hooks'
+import { useTranslation } from 'react-i18next'
+import React from 'react'
+//zukünftig in die einstellung eine möglichkeit zum cookie declinen einbauen
+//was passiert wenn accept geklickt wird?
```

Comment:
Für zukünftige ToDos bitte im GitHub Technical oder User Story Issues anlegen und ausformulieren.
Diese je nach Wichtigkeit im Backlog oder in der Icebox platzieren.

```suggestion
```